src/main/java/wolox/training/repositories/UsersRepository.java

```diff
@@ -18,7 +19,16 @@
 
     public Optional<User> findByUsername(String username);
 
-    List<User> findAllByBirthdateBetweenAndNameContainingIgnoreCase(
+    Optional<List<User>> findAllByBirthdateBetweenAndNameContainingIgnoreCase(
+            LocalDate startDate,
+            LocalDate endDate,
+            String name);
+
+    @Query("SELECT u FROM User u"
```

Comment:
This is good, can you add it documentation?, please

Reply:
corrected